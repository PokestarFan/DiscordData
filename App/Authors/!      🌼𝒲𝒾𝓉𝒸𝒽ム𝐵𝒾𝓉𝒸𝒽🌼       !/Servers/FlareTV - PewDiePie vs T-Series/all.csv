ID,Channel,Content Length,Content
558465768443084801,general,3,uwu
558466578048483330,general,9,K P O P ?
558466631223738377,general,20,I have been summoned
558466708244004892,general,7,Damn it
558466820517003266,general,41,*crawls back in the hellhole I came from*
558466868529070090,general,3,uwu
558467003334131749,general,17,I already do that
558467162398785566,general,9,I am hell
558467342191820800,general,35,<@558457743761997825> no_ I AM hell
558467663756525589,general,2,Ew
558467733851865098,general,28,Thats nastier than egg salad
558467790739079179,general,4,Wait
558467948969197589,general,60,Why do we have so many people named after food in this sever
558468046264729640,general,8,Probably
558468092653469697,general,4,Dog?
558468118738108417,general,14,Dog isn’t food
558468196202577920,general,19,Crab is also a good
558468258018230283,general,17,Food_ I mean food
558468323696836620,general,27,<@!380928993144340482> SAME
558468516924227595,general,50,<@!380928993144340482> I’ve never had crAwfish ;-;
558468576000868382,general,17,Hi bored_ I’m dad
558468680195768320,general,37,<@!380928993144340482> I just havrnts
558468757333213211,general,16,Stop flexing aaa
558468805743869998,general,39,<@!380928993144340482> I will then lmao
558469005073973268,general,39,<@!380928993144340482> I don’t think so
558469086317772830,general,30,*crawls out of hellhole* kPoP?
558469157608489020,general,27,<@558457743761997825> what?
558469381953159168,general,32,<@!380928993144340482> oooh okay
558469529177686017,general,34,Imma block eclipse if Chelsea does
558469548789989386,general,6,Oh rip
558469642855907328,general,34,Damn I needed a reason to be petty
558469747008733194,general,4,Yeah
558469812834140181,general,3,Ban
558470043776581632,general,3,Rip
558470153851895839,general,3,Yes
558470374392725517,general,11,Hit or miss
558470421343633408,general,27,I guess they never miss huh
558470443762450444,general,19,You got a boyfriend
558470476377096200,general,24,I bet he doesn’t kiss ya
558470519138287617,general,4,Muga
558470548598816778,general,7,PRIVATE
558470559483035668,general,7,YOU HOE
558470610825510922,general,32,Stop making me sound like a thot
558470708687011848,general,13,*no it’s not*
558470742623125517,general,2,NO
558470758737641485,general,4,Lmao
558470858683711488,general,25,Excuse me_ I’m a guy lmao
558471124900380693,general,17,Egg is not a girl
558471316299186186,general,2,he
558471329993719819,general,8,I’m a he
558471663167995914,general,33,<@!380928993144340482> crab is 10
558472219748204548,general,21,There’s a 10 year old
558473347428515841,general,0,
558778138558070795,general,4,Shit
558778153582198784,general,4,Fucc
558778191238660096,general,5,Bitsh
558778226692849664,general,3,Hoe
558798908881633290,general,3,uwu
558810331376386072,general,20,Aaaa I wanna join vc
558810365690118167,general,21,But I’m in another rn
558810390570729483,general,14,I can’t choose
558811900645736468,general,15,can i be banned
558811907855745024,general,3,pls
558812046381154305,general,6,i cant
558813039030304799,general,0,
558813202545377312,general,22,<@!176921245940580354>
558814011878014976,general,73,my milkshake brings all the boys to yard_ damn right is better than yours
558815672562483210,general,7,oh okay
558817008083402763,general,3,hoe
558818391738286121,general,25,family friendly porn_ wtf
558818749264691268,general,11,move me too
558818784387923974,general,20,wow i didnt even ask
558991746395013121,general,3,Wot
558992002637627393,general,2,Oh
558992235622825994,general,7,Oh nice
558992407476043795,general,3,oof
558992649881911316,general,20,Nothing personal kid
558992830853414964,general,19,Kid is not a virgin
558992953888997399,general,2,Oh
