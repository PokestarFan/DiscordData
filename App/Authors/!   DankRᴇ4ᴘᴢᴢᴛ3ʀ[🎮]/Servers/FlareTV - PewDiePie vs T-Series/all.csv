ID,Channel,Content Length,Content
558836432412737557,commands,8,pls rich
558836531054247969,commands,17,pls rob Dankchiko
558836838689800204,commands,5,t.shv
558836904477327390,commands,6,t.shav
558836949696118794,commands,4,tshv
558836961989885953,commands,5,t.shv
558837021498540052,commands,16,pls withdraw 500
558837181943250952,commands,5,t.shv
558837216588070923,commands,5,t.shv
558837249060372491,commands,4,tbal
558837310662246418,commands,5,t.bal
558837331272925190,commands,8,t.shovel
558837347169468452,commands,11,pls rob 10K
558837599213453312,commands,11,pls rob 10K
558837619681919010,commands,8,pls rich
558837671091240986,commands,26,pls profile the red dragon
558837736099020800,commands,8,t.shovel
558837763189899295,commands,22,pls rob the red dragon
558837780097007629,commands,5,t.shv
558837797671403530,commands,5,t.shv
558837837831864320,commands,22,pls rob the red dragon
558837862175473677,commands,8,t.shovel
558837888310312961,commands,5,t.shv
558837910829531165,commands,5,p!shv
558837958778814464,commands,5,p!bal
558837984406011925,commands,5,p!shv
558838018711224330,commands,5,p!shv
558838029297647654,commands,5,p!shv
558838055419772950,commands,5,p!shv
558838066958041091,commands,5,p!shv
558838078500765716,commands,5,p!shv
558838126466957342,commands,5,p!shv
558839353862914066,commands,16,pls withdraw 500
558839364604657664,commands,8,pls rich
558839384124817408,commands,5,p!shv
558839408736862212,commands,22,pls rob Dankchiko#0460
558839423383502860,commands,15,pls deposit all
558839452038856704,commands,11,hahaha late
558839510385819668,commands,20,btw tysm evasive lol
558839697254776854,commands,8,LOL REKT
558839717848678420,commands,17,pls withdraw 3000
558839741685039104,commands,36,pls share <@469172271651618826> 3000
558839769707184148,commands,7,hahhaha
558839788866764841,commands,8,nice lol
558839801596346378,commands,5,p!shv
558839836929425438,commands,5,p!shv
558839850246209542,commands,5,p!shv
558839887684567045,commands,5,p!shv
558839923734741032,commands,5,p!shv
558839950020313108,commands,5,p!shv
558839963605532734,commands,5,p!shv
558840012104531971,commands,5,p!bal
558840028231368719,commands,6,p!help
558840060049358858,commands,14,p!help economy
558840076239372289,commands,11,p!help shop
558840119931699245,commands,6,p!shop
558840196536467456,commands,16,pls withdraw 500
558840205696696330,commands,7,pls rob
558840260797399046,commands,6,p!help
558840276463124482,commands,5,p!shv
558840297883303947,commands,5,p!shv
558840320947650570,commands,5,p!shv
558840349762781195,commands,5,p!shv
558840364195381258,commands,5,p!shv
558840387763175432,commands,5,p!shv
558840405521727510,commands,5,p!shv
558840424601485324,commands,5,p!shv
558840437725462548,commands,5,p!shv
558840450497118248,commands,5,p!shv
558840479991463936,commands,5,p!shv
558840495351267329,commands,5,p!shv
558840506399064084,commands,5,p!shv
558840517417369630,commands,5,p!shv
558840524405080065,commands,5,p!shv
558840536778407936,commands,5,p!shv
558840544470761473,commands,5,p!shv
558840554281238561,commands,5,p!bal
558840774943309826,commands,17,pls withdraw 9999
558840788453425152,commands,12,pls bet 9999
558840800612450323,commands,6,im ded
558840809261105158,commands,7,pls bal
558840824654331934,commands,8,pls rich
558840853834104842,commands,22,pls rob the red dragon
558841069715062785,commands,16,pls withdraw all
558841086718771210,commands,11,pls bet all
558841112907874305,commands,3,bal
558841268935983118,commands,11,pls profile
558841454944976906,commands,12,t!slots 1000
558841543079886849,commands,11,t!slots 400
558841573798838276,commands,4,p!sh
558841587128467456,commands,3,p!s
558841608980791306,commands,3,p!s
558841619668008981,commands,4,p!sh
558841632145801226,commands,4,p!sh
558841644565266432,commands,4,p!sh
558841657211092992,commands,4,p!sh
558841666203549736,commands,4,p!sh
558841680581623818,commands,5,p!shv
558841700425138186,commands,5,p!bal
558841727285198878,commands,16,p!deposit all xd
558842039337484318,commands,5,p!shv
558842059461623808,commands,5,p!shv
558842080756105218,commands,5,p!shv
558842097558618132,commands,5,p!shv
558843454965415937,commands,8,pls rich
558843522942369813,commands,28,pls profile Julian Cruz#6021
558843540998848512,commands,11,pls rob 10K
558843556119183377,commands,18,pls deposit all ay
558843569893277707,commands,5,p!shv
558843584212631552,commands,5,p!shv
558843612683567125,commands,3,lol
558843647123128331,commands,4,p!sh
558843675644395520,commands,5,p!shv
558843720292761619,commands,5,p!shv
558843733261418529,commands,5,p!bal
558843785677897737,commands,8,t!credit
558843794670223360,commands,3,lol
558843872109920257,commands,9,t!profile
558843925599617025,commands,3,lol
558844319411339313,commands,7,pls rob
558844353364099103,commands,11,pls profile
558844396112707590,commands,16,ha im moar dan u
558844440182128640,commands,6,xD lol
558844481919778816,commands,16,not in money btw
558844490711040003,commands,3,lol
558844501628813352,commands,5,p!shv
558844522638082059,commands,7,p!steal
558844568368447488,commands,22,p!steal Dankchiko#0460
558844596763885568,commands,3,lol
558844614669238324,commands,5,p!shv
558844667471331348,commands,7,nvm lol
558844673901461536,commands,7,pls rob
558844684600868884,commands,8,pls rich
558844745317744677,commands,22,pls rob Winfred 3#2231
558844846060601346,commands,22,pls rob Winfred 3#2231
558844898363572261,commands,22,pls rob Winfred 3#2231
558844909591986178,commands,5,p!shv
558844923760083000,commands,5,p!sgv
558844930823290890,commands,5,p!shv
558844945587372045,commands,5,p!shv
558844964231184384,commands,3,lol
558845004504629268,commands,6,p!shop
558845937968218112,commands,7,pls bal
558845953390673922,commands,16,pls withdraw 500
558845964467699712,commands,8,pls rich
558845995367006209,commands,22,pls rob Winfred 3#2231
558846010059784196,commands,5,p!shv
558846036676706335,commands,5,p!shv
558846059833589779,commands,5,p!shv
558846070512418816,commands,5,p!shv
558846078112366593,commands,5,p!shv
558846088392605696,commands,5,p!shv
558846102875406342,commands,22,pls rob Winfred 3#2231
558846120361721860,commands,9,pls multi
558846144613187584,commands,22,pls rob Winfred 3#2231
558846180461772821,commands,22,pls rob Winfred 3#2231
558846200179064852,commands,19,pls deposit all yay
558846217958850560,commands,5,p!shv
558846252314394625,commands,3,lol
558846273806139408,commands,5,p!shv
558846313018687509,commands,26,remove sand from meh pls 😄
558846361651511296,commands,17,`pls remove sand`
558846397378461696,commands,25,im sure dat will work lol
558846440953217047,commands,5,emr y
558846512285614080,commands,15,pls deposit all
558846593072103435,commands,19,pls bet all pls boi
558846606334754826,commands,19,pls deposit all yay
558846632326594591,commands,7,pls bal
558846685619552275,commands,8,pls bank
558846700509331463,commands,7,pls inv
558846731014504460,commands,5,p!shv
558847377880907776,commands,5,p!shv
558847388148826151,commands,6,d!rank
558847398831456257,commands,5,!rank
558847420616933376,commands,3,LOL
558847453391093760,commands,9,pls guess
558847467307925504,commands,1,5
558847482101104646,commands,4,hint
558847509183725588,commands,1,3
558847575252533268,commands,43,just type 5 then hint so it would be easier
558847621142413342,commands,13,pls guess 100
558847642432700416,commands,2,28
558847656475099159,commands,2,39
558847671943823361,commands,4,hint
558847691971362817,commands,3,100
558847706915930122,commands,2,90
558847721654714405,commands,4,hint
558847741715808287,commands,2,98
558847762511167489,commands,2,95
558847772128837666,commands,4,hint
558847783994392578,commands,2,99
558847824280682512,commands,8,pls rich
558847848729411604,commands,16,pls withdraw 500
558847878014173204,commands,28,pls profile Julian Cruz#6021
558847892547305472,commands,24,pls rob Julian Cruz#6021
558847929427689482,commands,38,pls share <@494683343938519071> 750 ol
558847942060933121,commands,15,pls deposit all
558847988454260738,commands,3,rip
558848020372914186,commands,17,pls withdraw 5000
558848032775471116,commands,13,pls slots all
558848061678288906,commands,34,pls deposit all look like im lucky
558848077197344769,commands,7,pls bal
558848094947508225,commands,8,pls rich
558848147418382346,commands,33,pls profile <@556251957162016799>
558848171401543700,commands,36,pls share <@556251957162016799> 4000
558848217144492053,commands,25,oml ur losing so much lol
558848274501599243,commands,7,finnaly
558848612868685834,commands,25,lol <@544744541555654657>
558848628962099240,commands,16,pls withdraw 500
558848637313089556,commands,8,pls rich
558848675934240808,commands,28,pls rob Cupcakegirl9012#0394
558848696545050626,commands,2,gl
558848731714289670,commands,32,pls profile Cupcakegirl9012#0394
558848739200991237,commands,3,rip
558848752597729284,commands,28,pls rob Cupcakegirl9012#0394
558848767844155412,commands,17,pls withdraw 9999
558848782889123840,commands,12,pls bet 9999
558848796298182676,commands,7,rip lol
558848827680096257,commands,3,oml
558848835963715585,commands,7,pls bal
558848858210435075,commands,15,pls deposit all
558848868679286790,commands,17,pls withdraw 9999
558848890288209949,commands,39,pls bet all pls pls pls pls pls pls pls
558848903424770058,commands,18,pls deposit all ty
558848913721786373,commands,7,pls rob
558848922525761539,commands,5,p!shv
558849943788584973,commands,8,pls rich
558849966856994834,commands,28,pls rob Cupcakegirl9012#0394
558849984171212824,commands,16,pls withdraw 500
558849990546423824,commands,28,pls rob Cupcakegirl9012#0394
558850012436496416,commands,5,reeee
558850024939978762,commands,7,pls bal
558850050109997056,commands,14,pls worklist 3
558850103511613460,commands,46,<@556251957162016799> lol u r gonna get robbed
558850123283562496,commands,26,y dont u get a padlock den
558850171014742016,commands,17,pls withdraw 5000
558850181890703393,commands,14,pls slots 5000
558850194343460864,commands,9,reeee lol
558850252489097217,commands,33,pls rpofile <@279807910782173184>
558850281706881045,commands,33,pls profile <@279807910782173184>
558850310546784259,commands,30,so y are u donating to den LOL
558850332885778432,commands,19,get fking rekt lmao
558850416524263434,commands,17,= real currency 😄
558850466088484865,commands,20,cuz ur rich? huh lol
558850489689702401,commands,3,LOL
558850564880859146,commands,3,lol
558850596837523456,commands,21,y r u using nsfw here
558850662205751296,commands,16,pls withdraw 100
558850669541457928,commands,11,pls lottery
558850678798417940,commands,3,yes
558850688663289867,commands,15,pls deposit all
558851127945199627,commands,7,pls rob
558851144529608724,commands,17,pls withdraw 9999
558851157347270657,commands,15,pls gamble 9999
558851175957397522,commands,49,reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
558851185436786721,commands,7,pls bal
558851196689973270,commands,8,pls rank
558851213739687954,commands,5,!rnak
558851222774480927,commands,10,!rank bruh
558851236103716875,commands,5,!rank
558851257473826817,commands,5,breah
558851271503642639,commands,17,pls withdraw 9999
558851295486803971,commands,36,pls bet 9999 last one pls let me win
558851309139132426,commands,4,....
558851324444278815,commands,7,pls bal
558851340529303556,commands,16,pls withdraw 500
558851347571671052,commands,7,pls rob
558851366034866196,commands,8,pls rich
558851379783925790,commands,28,pls rob Cupcakegirl9012#0394
558851451972091957,commands,24,pls rob Shadow4DayZ#6757
558851471031140353,commands,4,brah
558851964784607244,commands,16,pls withdraw 500
558851975408779324,commands,7,pls rob
558852013274693673,commands,16,pls withdraw all
558852021961359361,commands,7,pls bal
558852040953167874,commands,12,pls bet 1067
558852055523917835,commands,11,reeeeeeeeee
558852072615968788,commands,7,pls bal
558852092673130507,commands,3,lol
558852113908760576,commands,24,pls rob Shadow4DayZ#6757
558852554965123082,commands,24,pls rob Shadow4DayZ#6757
558852648137392129,commands,24,pls rob Shadow4DayZ#6757
558852773958123523,commands,24,pls rob Shadow4DayZ#6757
558852791301570582,commands,7,pls bal
558852806631489537,commands,3,lol
558852843730108440,commands,3,rip
558852900454006784,commands,3,uwu
558852926853087242,commands,3,owo
558852961330135042,commands,3,lol
558853383067533313,commands,10,pls search
560286113684520961,commands,27,pls profile Evade Hype#4338
560286165933228033,commands,23,pls rob Evade Hype#4338
560286199252516876,commands,26,pls rob 🍅M3M3F!ND3R Bason🍅
560286216109555722,commands,15,pls deposit all
560286236812771339,commands,8,pls rich
560286830789132293,commands,7,pls rob
560287789615153162,commands,8,pls rich
560287814701416458,commands,20,pls profile CornBlox
560287830002368518,commands,16,pls withdraw 500
560287861711306781,commands,16,pls rob CornBlox
560287876219273227,commands,15,pls deposit all
560287897509691422,commands,7,pls bal
560287919428861976,commands,17,pls withdraw 9999
560287937712095255,commands,31,pls bet all pls pls pls pls pls
560287953440473113,commands,23,reeeeeeeeeeeeeeeeeeeeee
