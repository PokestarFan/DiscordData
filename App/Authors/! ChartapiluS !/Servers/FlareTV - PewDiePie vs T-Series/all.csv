ID,Channel,Content Length,Content
558989902629437442,pewdiepie-vs-tseries,21,like a alien speaking
558990353827627029,pewdiepie-vs-tseries,24,biar blajar english h3h3
558991108256956441,pewdiepie-vs-tseries,45,can you go to heaven from say "sub to pewd"??
558991748622319626,pewdiepie-vs-tseries,6,hiyaaa
558991892927479829,pewdiepie-vs-tseries,36,weebs detected <@407076961786134539>
558992074595369012,pewdiepie-vs-tseries,24,B A C O T     A N Y I NG
558993195791548429,pewdiepie-vs-tseries,25,but why the subs so many?
558993211767652361,pewdiepie-vs-tseries,9,like pewd
558993284425318413,pewdiepie-vs-tseries,20,its why i hate t-gay
558994940743385088,pewdiepie-vs-tseries,29,Ice cream = I scream = ScreaM
558995657680224257,pewdiepie-vs-tseries,38,DMC 5 has great graphic when battle XD
558995709081419797,pewdiepie-vs-tseries,18,but i got lag lmao
558996210283839498,general,37,<@544111003894415362> hee anata dare?
558996232836612126,general,3,wa*
558996407831232513,memes,8,pls meme
558996508167634959,memes,3,nah
558996607526502400,memes,27,dat memes dont get me laugh
558996661418852372,pewdiepie-vs-tseries,13,what time is?
558996952490967070,pewdiepie-vs-tseries,33,<@508486121798238209> RIP english
558997010368167948,pewdiepie-vs-tseries,29,dont say english when u can't
558997106996543488,pewdiepie-vs-tseries,21,its indonesian people
558997127016218658,pewdiepie-vs-tseries,4,h3h3
558997230183514112,pewdiepie-vs-tseries,29,T-series has left the server*
558997313968668672,pewdiepie-vs-tseries,44,<@558992002323316737> no one bullying u lmao
