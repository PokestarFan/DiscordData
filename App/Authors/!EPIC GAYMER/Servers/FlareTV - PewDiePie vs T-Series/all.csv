ID,Channel,Content Length,Content
558792470918332432,subcount,1,_
