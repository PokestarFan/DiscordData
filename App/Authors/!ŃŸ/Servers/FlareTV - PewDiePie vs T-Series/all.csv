ID,Channel,Content Length,Content
558443386189905920,general,3,Ooh
558443483380318219,general,6,Heller
558443617589526528,general,3,Oof
558444660264009768,general,2,Of
558444760461737995,general,4,Hmmm
558444778887577636,general,1,👀
558445276545679390,general,3,Oof
558445305276661771,general,35,<:kermitragegun:492896354440314880>
558445339871412224,general,26,<:True:529561303103307776>
558445399187128322,general,18,Wow no staff is on
558445419177181186,general,8,Love hat
558445539109371934,general,3,Wut
558446949695750144,general,3,Oof
558447069061185546,commands,5,!rank
558447084831768595,commands,10,Pls search
558447089986830336,commands,7,Pls beg
558447100405350433,commands,15,Pls deposit all
558453776697262127,general,3,Oof
558453854186766336,general,3,👀👀👀
558453865859776512,general,33,<a:peekstitch:541444790349332510>
558453890891251723,general,6,Try it
558453900869500938,general,2,Hi
558453942443573248,general,4,Good
558454001616814094,general,4,Cool
558454045438640128,commands,6,Defend
558454082281668608,commands,3,...
558454093568278539,commands,5,Punch
558454140632825876,commands,5,Punch
558454179266560026,commands,3,WOW
558454205803921409,commands,31,Pls kill <@!341648742685999106>
558454261705605120,commands,5,Lmfao
558454328814338069,general,27,<@!518924074198761482> what
558454396514467840,general,2,Ok
558454542132445214,general,52,Then I’ll call you ?mute <@!518924074198761482> 1200
558454720067272705,general,42,<@!341648742685999106> who are you subbing
558454816389464095,general,8,Stopping
558454827261362196,general,7,STOOPID
558454869334294578,general,27,<@!518924074198761482> Baku
558454979745284106,general,4,Baku
558455014302023731,general,5,Oh ok
558455044047896609,general,37,I didn’t know if this was Baku or not
558455092706017313,general,3,Lol
558455177347334155,commands,5,!rank
558455215234220032,general,11,That’s tuff
558455261925212160,general,3,Yes
558455297337851916,general,6,That’s
558455301905580052,general,4,Tuff
558455311091105792,general,5,Bitch
558455529534521356,general,3,Lol
558455689324920852,general,100,<a:monkaShoot:444921483802705920> <:DeadChat:524896170787733514> <:kermitragegun:492896354440314880>
558456044083347457,general,3,Oof
558456198794313730,general,3,Ooh
558456540508717057,general,3,Oof
558456690568331274,general,3,Oof
558457142013853716,general,4,Reee
558457234896584704,general,2,Oo
558457247521570816,general,100,<:kermitragegun:492896354440314880> <:MonkaGun:532226138110689280> <a:monkaShoot:444921483802705920>
558457561372819456,general,30,<:DeadChat:524896170787733514>
558457883520663570,general,4,No u
558457922393341974,general,0,
558458090895310848,general,28,<@339486831991062529> okokok
558458602881417226,general,3,Oof
558458700365430824,general,34,Well aren’t you captain save a hoe
558458967760830464,general,3,Oof
558459275618549802,general,3,Oof
558459502492647424,general,2,Hi
558459530900537375,general,28,<a:yoshi:553637790311776256>
558460057201803284,general,3,Rip
558460522274488320,general,25,<:Wut:519226309059149833>
558461102006992917,general,98,Does anyone else ever just get flashes of anger out of no where and want to take it out on someone
558461206361538580,general,2,Ok
558461260430180430,general,60,Well that’s happening rn and I want to ban the entire server
558461265165418526,general,3,Soo
558461288242610176,general,14,?ban @everyone
558461477330223114,general,96,<@!380928993144340482> my thigh hurts. I think I pulled it. How I pulled it tho is a funny story
558461633605926912,general,2,Ok
558461643298963467,general,3,DMs
558465461092876318,general,3,Wut
558465740030738462,general,3,Oof
558467056878485504,general,3,Rip
558467071080398868,general,3,Wtf
558468913915232257,general,11,Dasss a rip
558469091833282562,general,3,Oof
558469694290526244,general,76,If I mute Chelsea I expect she stays muted<a:eyesshaking:527915095892688896>
558469858174566422,general,6,No one
558469865023864832,general,5,CARES
558469956350771233,general,9,Yes I did
558469972381138947,general,20,And I’ll do it again
558470008611536906,general,4,Jkjk
558470749480812554,general,1,O
558470819164979217,commands,5,!rank
558470833367154748,commands,10,Pls search
558470837875769346,commands,7,Pls beg
558470844309831680,commands,15,Pls deposit all
558471035054194698,general,56,<@!449951831720853523> you lost vc privileges for a week
558471155619725322,general,49,“There’s a spic downstairs I’m gonna go kill him”
558471278520958986,general,42,You’re either dumb or ignorant of the word
558471441587240980,general,103,Spic is short for Hispanic it’s what gm white people called us. It’s like the n word for Spanish people
558471589331468289,general,4,Dies
558471807796248579,general,27,<@390924141697564675> Dewit
558471837034479617,general,5,Never
558472172381798433,general,3,Why
558472313083789322,general,26,?ban <@558457743761997825>
558472484031037440,general,82,You’re right <@!449951831720853523> I should’ve banned but you’re lucky i like you
558472595159121931,general,9,Hennyways
558473926003523594,general,3,Oof
558757771990925320,general,3,Oof
558757922772090901,general,3,Oof
558760269296435200,general,3,Oof
558761602233532426,commands,10,Pls search
558761607820345347,commands,7,Pls beg
558761619954597898,commands,11,Pls deposit
558763488844185629,commands,3,Pls
558763574667771904,general,3,Wtf
558765143811555449,general,2,Oo
558766344397651968,general,33,<a:peekstitch:541444790349332510>
558768548173053984,commands,10,Pls search
558768566791569408,commands,7,Pls beg
558768573561045002,commands,15,Pls deposit all
558772710466125844,commands,10,Pls search
558772717235470353,commands,7,Pls beg
558772727306256401,commands,15,Pls deposit all
558772868071030811,general,3,Wut
558772882516213771,general,28,<a:yoshi:553637790311776256>
558772973209780225,general,9,Hennyways
558777697732395049,general,3,Oof
558781588700397588,general,3,Oof
558785599109332992,general,6,Lmfaoo
558785638237732865,general,10,I’m peeing
558787458431582209,general,12,Weeepwooooop
558787668163428353,general,31,<a:headtilt:486919048643215422>
558790368745750538,general,3,Oof
558797042768543765,general,2,Ok
558807799262347264,general,3,Oof
558812677879758869,general,5,Lmfao
558816365641990174,general,4,What
558820156030058496,general,4,Ooof
558820251597406208,general,42,<@556251957162016799> Whyd you say my name
558820588253347850,general,3,Oof
558822646058319872,general,14,I have too pee
558822846047059998,general,2,Ok
558823020186173461,general,3,Sad
558870918521290792,general,4,....
558871174965231626,general,125,?mute <@!449951831720853523> 60 that’s 2 weeks w/ no vc privileges since you wanted to find a way to be unmuted and undeafend
558871607679123456,general,32,?warn <@516951840550682625> spam
558872581474615338,general,3,Oof
559060292290478082,pewdiepie-vs-tseries,31,?mute <@558468497718640640> 100
559060302721843200,pewdiepie-vs-tseries,5,Muted
559060697086951446,general,198,<@390924141697564675> it would’ve been better if you handled this situation better and accepted that you weren’t picked this time instead of bitching about it. This just shows you’re to immature🤷‍♂️
559060805203656715,general,28,<@!398320585903702018> shhhh
559071178027827202,commands,10,Pls search
559071184252305429,commands,7,Pls beg
559071191256793098,commands,15,Pls deposit all
560601986173763585,general,3,Oof
560620828887220224,general,25,You guys annoy me so much
560635277102809108,general,4,Poof
560652082869305376,general,3,Oof
560652332979847178,general,52,<:WTFW:543143882997760051> <:Wut:519226309059149833>
560652455851982849,general,36,<:PichuSurprised:552530445686800395>
560652509438541836,commands,5,!rank
560652525410451467,commands,10,Pls search
560652531156516869,commands,7,Pls beg
560652543538102274,commands,8,Pls work
560652788917469196,commands,25,<:Wut:519226309059149833>
560652799617007637,commands,3,Oof
560652891396767754,commands,7,For now
560652957276962830,commands,20,Eventually you won’t
560652976432087044,commands,16,I stopped caring
560653024490684440,commands,35,I stoppped caring about this server
560653141377548288,general,3,Oof
560653158393708545,general,4,K Gn
560653271644241981,general,56,When no one says good night back to me. What a great day
560975931536179202,general,3,Oof
560976003497721877,general,5,MuTeD
560976086494609409,general,1,K
560976238882193419,general,2,Kk
560976294473629723,general,26,?ban <@555864334048296964>
560976336202760228,general,4,YaWn
560976443962556426,general,84,You guys love aggravating me with your little shade throwing like I don’t realize it
560976601446219776,general,69,And when I say something “OmG sToP bEiNg such an ass to me and my gf”
560976696858116116,general,17,Hennyways I’m out
560976896888668171,general,113,<@431045920688177152> I was mimicking wat someone was saying please don’t get banned by not minding your business
560976961565097994,general,40,<@!285835383374348289> something to say?
560977000895086642,general,5,Oh ok
560977086836113428,general,4,Haii
560977154561802260,general,9,Who’s dad
560977186308489226,general,3,Oof
560977213458087956,general,18,No my dad is Dylan
560977222631161856,general,9,Try again
560977452331958272,general,25,<:Wut:519226309059149833>
560977864917516299,commands,8,Pls work
560977891794354186,commands,7,Channel
560977901579927555,commands,15,Pls deposit all
560978861601456138,general,2,Ok
560978939623768084,general,1,K
560979088022568962,general,68,Maybe if things get really bad for you_ you can get back with hoodie
560979099011514399,general,3,OOF
560979125616246805,general,32,But things have to be really bad
560979228615507968,general,2,Ew
560979380147322883,general,30,He did the same to me at first
560979392743079936,general,7,For you
560979435092967436,general,30,You lose them how you get them
560979597722648599,general,79,Maybe we should just team up together and murder his entire family tree of dogs
560979932419850250,commands,5,!rank
560980110308802591,general,3,Oof
560981189121409064,commands,10,Pls search
560981193458188291,commands,7,Pls beg
560981200567795744,commands,15,Pls deposit all
560981727619579943,general,7,Okizzle
560981804371410965,general,5,DELLA
560981839293186048,general,2,Hį
560981974492381184,general,39,I’ve been in a bad mood for a long time
560981986601074700,general,7,Because
560982014497390622,general,25,I can’t take naps anymore
560982070977888276,general,4,Work
560982079655903252,general,4,Work
560982089273573416,general,10,And school
560982110635032616,general,14,Plus more work
560982155040129025,general,3,Yes
560982180474388520,general,8,Hospital
560984963344891904,general,3,YES
560985052037644288,commands,28,!rank <@!188110017944223754>
560985070362689546,commands,4,OMFG
560985078507765760,commands,8,HURRY UP
560985197789839391,general,3,Yay
561000433326227457,general,6,I peed
561000566365487124,general,13,Is it Sunday?
561000669620731915,general,90,<@!325122999789551616> Once...Again do not bring private in a vc until the end of the week
561000691229786152,general,90,<@!325122999789551616> Once...Again do not bring private in a vc until the end of the week
561000700981805056,general,90,<@!325122999789551616> Once...Again do not bring private in a vc until the end of the week
561000707713400842,general,90,<@!325122999789551616> Once...Again do not bring private in a vc until the end of the week
561000817876795392,general,66,I mean I’ve said it so many times but people continue to ignore me
561002875644280842,general,5,Dewit
561002890706026496,general,10,Then leave
561002898251579392,general,3,Lol
561003452319268896,general,13,I have to pee
561003504332832779,general,5,Dewit
561003553603321878,general,34,<@!325122999789551616> wanna fight
561003565678592024,general,115,<a:yoshi:553637790311776256> <a:yoshi:553637790311776256> <a:yoshi:553637790311776256> <a:yoshi:553637790311776256>
561003604878688266,general,34,I’ll have thicc yoshi eat your ass
561003659370954775,general,5,NOOOO
561003691461574657,general,23,I’ll piss in your mouth
561003713066696734,general,3,Yes
561003726840791040,general,13,Golden shower
561003765466136586,general,23,He doesn’t have to know
561003809002749954,general,14,But he doesn’t
561003839436881932,general,3,Not
