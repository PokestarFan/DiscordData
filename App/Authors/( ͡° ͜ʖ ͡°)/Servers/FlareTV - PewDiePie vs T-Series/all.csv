ID,Channel,Content Length,Content
559106126273183764,general,18,My Berryâ€™s are red
