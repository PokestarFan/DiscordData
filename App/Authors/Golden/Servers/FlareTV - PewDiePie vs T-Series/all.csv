ID,Channel,Content Length,Content
560653879788830732,general,7,Remebet
560653914249101332,general,10,We can win
560654206491688980,pewdiepie-vs-tseries,32,They are fucking 1000 subs ahead
560654256739450900,pewdiepie-vs-tseries,7,And how
560654611191431210,pewdiepie-vs-tseries,22,Join public lobby vc 2
